tests/python/gaia-ui-tests/gaiatest/apps/messages/app.py

```diff
@@ -11,6 +11,7 @@
 class Messages(Base):
 
     name = 'Messages'
+    manifest_url = "app://sms.gaiamobile.org/manifest.webapp"
```

Comment:
This would need the  self.DEFAULT_PROTOCOL and self.DEFAULT_APP_HOSTNAME treatment, I would think.